plugins/bazaar-backend/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@internal/plugin-bazaar-backend",
```

Comment:
Probably want to give this a better name, you can do it as `@backstage/plugin-bazaar-backend` rather than `@internal` to publish it.